effortless-chef-io/variables-and-config.md

```diff
@@ -1,105 +1,131 @@
-# Plan Variables and Habitat Configurations
+# Plan Variables and Chef Habitat Configurations
 
-This documents the options for both your plan and your habitat configurations file.
+This documents the options for both your plan and your `habitat` configuration file.
 
 ## Effortless Config
 
-### Plan Variables
+### Effortless Config Plan Variables
 
-You can use all the default plan variables that are shipped with habitat you cand find a list of these [here](https://www.habitat.sh/docs/reference/#plan-variables).
+You can use all the default plan variables shipped with Chef Habitat. Read more about [plan variables](https://www.habitat.sh/docs/reference/#plan-variables) in the Chef Habitat documentation.
 
-**$scaffold_chef_client**: Variable to change the chef-client habitat package that is used. Default is `chef/chef-infra-client`
+scaffold_chef_client
+: The Chef Habitat `chef-infra-client` package used. Change to use a different package. Default is `chef/chef-infra-client`
 
-**$scaffold_cacerts**: Variable to change the CACerts habitat package that is used during the chef-client run. Default is `chef/cacerts`
+scaffold_cacerts
+: The Chef Habitat `cacerts` package used during the Chef Infra Client run. Change to use a different package.Default is `chef/cacerts`
 
-**$scaffold_policyfile_path**: Path to the policyfile to be built. Default is `$PLAN_CONTEXT/../policyfiles`
+scaffold_policyfile_path
+: Path to the policyfile. Default is `$PLAN_CONTEXT/../policyfiles`
 
-**$scaffold_data_bags_path**: Path to the data_bags folder containing data_bags that the cookbook may need to run. Default is `$PLAN_CONTEXT/../data_bags`
+scaffold_data_bags_path
+: Path to the `data_bags` directory that contains the `data_bags`, if needed by your cookbook. Default is `$PLAN_CONTEXT/../data_bags`
 
-### Habitat Config Settings
+### Effortless Config Chef Habitat Settings
 
-**interval**: Interval the chef-client with run at. Default is 1800 seconds (30 minutes).
+interval
+: Frequency of how often in seconds that the Chef Infra Client runs. Default is `1800`
 
-**splay**: Number that will be used as the last number in a range to create a random number that will be added to the interval to determine when the chef-client is run again. Default is 1800 seconds.
+splay
+: A randomly generated sleep time value in seconds added to the interval and helps determine when the chef-client runs again. Default is `1800`
```

Comment:
@TheLunaticScripter Is this modified description for "splay" accurate?

Reply:
Yep that looks great.